src/modules/reporting/selectors/select-dispute-markets.js

```diff
@@ -41,6 +42,25 @@ export const selectMarketsInDispute = createSelector(
         nonPotentialForkingMarkets.push(market)
       }
     })
+
+    // Sort disputed markets by: 1) dispute round, and 2) highest percent staked in non-tentative-winning outcome
+    Object.keys(nonPotentialForkingMarkets).forEach((marketKey) => {
+      if (nonPotentialForkingMarkets[marketKey].disputeInfo) {
+        nonPotentialForkingMarkets[marketKey].disputeInfo.highestPercentStaked = createBigNumber(0)
+        Object.keys(nonPotentialForkingMarkets[marketKey].disputeInfo.stakes).forEach((stakeKey) => {
+          if (!nonPotentialForkingMarkets[marketKey].disputeInfo.stakes[stakeKey].tentativeWinning) {
+            const percentStakedInOutcome = createBigNumber(nonPotentialForkingMarkets[marketKey].disputeInfo.stakes[stakeKey].stakeCurrent)
+              .div(createBigNumber(nonPotentialForkingMarkets[marketKey].disputeInfo.stakes[stakeKey].bondSizeCurrent))
+            if (percentStakedInOutcome.gt(nonPotentialForkingMarkets[marketKey].disputeInfo.highestPercentStaked)) {
+              nonPotentialForkingMarkets[marketKey].disputeInfo.highestPercentStaked = percentStakedInOutcome
+            }
+          }
+          nonPotentialForkingMarkets[marketKey].disputeInfo.highestPercentStaked = nonPotentialForkingMarkets[marketKey].disputeInfo.highestPercentStaked.toString()
+        })
+      }
+    })
+    nonPotentialForkingMarkets = orderBy(nonPotentialForkingMarkets, ['disputeInfo.disputeRound', 'disputeInfo.highestPercentStaked'], ['desc', 'desc'])
```

Comment:
shouldn't this be sorting by higher rounds first?

Reply:
<img width="1623" alt="screen shot 2018-07-19 at 8 19 49 am" src="https://user-images.githubusercontent.com/6775839/42952198-cbb3e0d0-8b2c-11e8-844f-fc8ac72ab98c.png">

Reply:
Yes, it should be. I had forgotten to test this section of the page. Will update the selector for this as well.